addons/my-ocular/addon.json

```diff
@@ -1,15 +1,16 @@
 {
-  "name": "ocular integration",
+  "name": "Ocular integration",
```

Comment:
`ocular` (all lowercase) is the official name, I'm pretty sure it shouldn't be capitalized.
```suggestion
  "name": "ocular integration",
```

Reply:
But it looks better? @jeffalo what do you think?

Reply:
ocular is spelt ocular.